README.md

```diff
@@ -171,8 +172,10 @@ jobs:
       - run: npx semantic-release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          # In case you want to publish to Visual Studio Marketplace
+          # In case you want to publish to Visual Studio Marketplace.  Remove if using VSCE_AZURE_CREDENTIALS
           VSCE_PAT: ${{ secrets.VSCE_PAT }}
+          # Set in order to publish to Visual Studio Marketplace using Azure CLI credentials
+          #VSCE_AZURE_CREDENTIALS: true
```

Comment:
I'd rather omit this. If users want to use it, they will know how to use it. There's even your example below.

What do you think?